amplify-tools-gradle-plugin/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="app_name">amplifytools</string>
```

Comment:
Do you use this anywhere? If not, just delete `src/main/res` entirely from the project.